custom_components/tornado/__init__.py

```
@@ -7,19 +7,31 @@
 from .aux_cloud import AuxCloudAPI
 from .const import CONF_EMAIL, CONF_PASSWORD, CONF_REGION, DOMAIN
 
 if TYPE_CHECKING:
     from homeassistant.config_entries import ConfigEntry
-    from homeassistant.core import HomeAssistant
 
 PLATFORMS: list[Platform] = [Platform.CLIMATE]
 _LOGGER = logging.getLogger(__name__)
 
 
+async def async_setup(hass: HomeAssistant, config: dict) -> bool:
```

Ruff (0.8.2)

Unused function argument: `config`

(ARG001)